Adds `astro add` docs to integrations (#258)
withastro/docs: src/pages/en/guides/integrations-guide.md

In this example, we will add the `@astrojs/react` integration to add React support to your Astro project. The process for adding any other framwork (Preact, Vue, Svelte or Solid.js) is almost identical and can be followed using the same steps outlined below.

> > Feeling adventurous? Astro recent launched an **experimental** `astro add` command to automate this process! If you'd like to try it, just run `astro add react`. That's it!

Reviewer: Nit: should we say `npx astro add` in case the astro CLI isn't configured as a global? I know this is a windows issue in particular.

```suggestion
> Feeling adventurous? Astro recent launched an **experimental** `npx astro add` command to automate this process! If you'd like to try it, just run `astro add react`. That's it!
```

> > Feeling adventurous? Astro recent launched an **experimental** `astro add` command to automate this process! If you'd like to try it, just run `astro add react`. That's it!

Reviewer 2: ```suggestion
> Feeling adventurous? Astro recent launched an **experimental** `astro add` command to automate this process! Instead of the steps below, you can just run `npx astro add react`. That's it! Skip down to [Automated Integration Setup](/en/guides/integrations-guide/#automatic-integration-setup) for more details.
```

Nit nit: I love this call-out, and the wording itself is perfect, but when I look at it in the preview, visually it doesn't scream "Stop!" to me and isn't separated enough from  "First, you will need to.." instructions. 

So, the above is just a suggestion you can take, leave or improve upon, but I would make sure to look at this in the deploy preview for confirmation that it's visually clear, as well. (You could even consider an emoji to start the quote block.)

Author: Appreciate the thoughts! I'll see if there's a way to make this stand out more ☺️

I do love your wording suggestions, though. Added those!

- ✅ called your integration as a function (`[react()]`, not `[react]`)
- ✅ removed the deprecated `renderers:` configuration

## Automatic Integration Setup

Astro recent launched an **experimental** `astro add` command to automate the setup of integrations.

> > We will always ask for confirmation before updating any of your files, but it never hurts to have a version controlled backup just in case.

Reviewer 2: ```suggestion
> We will always ask for confirmation before updating any of your files, but it never hurts to have a version-controlled backup just in case.
```

Instead of the manual configuration outlined above, just run `astro add [name]` and our automatic integration wizard will update your configuration file and install any necessary dependencies.

```shell
# Using NPM
npx astro add react
# Using Yarn
yarn astro add react
# Using PNPM
pnpx astro add react
```

> It's even possible to configure multiple integrations at the same time!

Reviewer: It just gets better and better 🤯
